src/Polar/Constraints/line_flow.jl

```diff
@@ -74,32 +74,25 @@ function adjoint!(
     wait(ev)
 end
 
-function matpower_jacobian(polar::PolarForm, ::State, ::typeof(flow_constraints), V)
+function matpower_jacobian(polar::PolarForm, X::Union{State,Control}, ::typeof(flow_constraints), V)
```

Comment:
@frapac Does this mean that this function definition is for both State and Control?

X::Union{State,Control}

Reply:
Indeed: we could use this function either with `State` and `Control`. The reason why I am doing that, instead of coding two functions
```julia
matpower_jacobian(..., X::State, ...)
matpower_jacobian(..., X::Control, ...)
```
is to avoid duplicate code, as the implementations of the two functions would look almost similar. 
Performance wise, the compiler is clever enough to handle the switch efficiently (I have a reference about that, let me share it with you)